Packs/FlashpointFeed/Integrations/FlashpointFeed/FlashpointFeed.yml

```diff
@@ -0,0 +1,187 @@
+  required: true
+  type: 4
+- additionalinfo: 'Supports multiple values such as url, domain, ip-src. Supports
+    comma separated values. If not specified, it fetches all the indicators. See all
+    available types: https://www.circl.lu/doc/misp/categories-and-types/#types.'
+  display: Types of the indicators to fetch
+  hidden: false
+  name: types
+  required: false
+  type: 0
+- additionalinfo: 'Backfill indicators by providing date or relative timestamp. (Formats
+    accepted: 2 minutes, 2 hours, 2 days, 2 weeks, 2 months, 2 years, yyyy-mm-dd,
+    yyyy-mm-ddTHH:MM:SSZ, etc)'
+  defaultvalue: 3 days
+  display: First fetch time interval
+  name: first_fetch
+  required: false
+  type: 0
+- defaultvalue: 'true'
+  display: Fetch indicators
+  name: feed
+  required: false
+  type: 8
+- additionalinfo: Indicators from this integration instance will be marked with this
+    reputation
+  defaultvalue: Bad
+  display: Indicator Reputation
+  name: feedReputation
+  options:
+  - None
+  - Good
+  - Suspicious
+  - Bad
+  required: false
+  type: 18
+- additionalinfo: Reliability of the source providing the intelligence data
+  display: Source Reliability
+  name: feedReliability
+  options:
+  - A - Completely reliable
+  - B - Usually reliable
+  - C - Fairly reliable
+  - D - Not usually reliable
+  - E - Unreliable
+  - F - Reliability cannot be judged
+  required: true
+  type: 15
+- display: ''
+  name: feedExpirationPolicy
+  options:
+  - never
+  - interval
+  - indicatorType
+  - suddenDeath
+  required: false
+  type: 17
+- display: ''
+  defaultvalue: '20160'
+  name: feedExpirationInterval
+  required: false
+  type: 1
+- defaultvalue: '240'
+  display: Feed Fetch Interval
+  name: feedFetchInterval
+  required: false
+  type: 19
+- additionalinfo: When selected, the exclusion list is ignored for indicators from
+    this feed. This means that if an indicator from this feed is on the exclusion
+    list, the indicator might still be added to the system.
+  display: Bypass exclusion list
+  name: feedBypassExclusionList
+  required: false
+  type: 8
+- additionalinfo: Supports CSV values.
+  display: Tags
+  name: feedTags
+  required: false
+  type: 0
+- additionalinfo: The Traffic Light Protocol (TLP) designation to apply to indicators
+    fetched from the feed
+  display: Traffic Light Protocol Color
+  name: tlp_color
+  options:
+  - RED
+  - AMBER
+  - GREEN
+  - WHITE
+  required: false
+  type: 15
+- additionalinfo: To indicate to the Cortex XSOAR server that a feed is incremental.
+    Generally feeds that fetch based on a time range. For example, a daily feed which
+    provides new indicators for the last day or a feed which is immutable and provides
+    indicators from a search date onwards.
+  defaultvalue: 'true'
+  hidden: true
+  name: feedIncremental
+  required: false
+  type: 8
+- additionalinfo: Create relationships between indicators as part of Enrichment.
+  defaultvalue: 'false'
+  display: Relationship
```

Comment:
This name should possibly be improved
@ChanochShayner do you think it's clear enough?

Reply:
Same as above.